openrtb_ext/convert_down.go

```diff
@@ -177,3 +178,122 @@ func moveRewardedFrom26ToPrebidExt(i *ImpWrapper) error {
 
 	return nil
 }
+
+// clear26Fields sets all fields introduced in OpenRTB 2.6 to default values, which
+// will cause them to omitted during json marshal.
+func clear26Fields(r *RequestWrapper) {
+	r.WLangB = nil
+	r.CatTax = 0
+
+	if app := r.App; app != nil {
```

Comment:
Is there a reason for this construction (`app := r.App`), rather than just checking if `r.App` is nil, and referencing `r.App` rather than `app`? Is it just to save the two characters `r.`?

Reply:
It's to save on characters, and likely performance. while `r.App` isn't a big deal, `r.App.Content.Producer` starts to become a bit silly.